src/ucp/api/ucp.h

```diff
@@ -2457,6 +2508,44 @@ ucs_status_ptr_t ucp_tag_send_sync_nb(ucp_ep_h ep, const void *buffer, size_t co
                                       ucp_send_callback_t cb);
 
 
+/**
+ * @ingroup UCP_COMM
+ * @brief Non-blocking tagged-send operations
+ *
+ * This routine sends a messages that is described by the local address @a
+ * buffer, size @a count object to the destination endpoint @a ep. Each
+ * message is associated with a @a tag value that is used for message
+ * matching on the @ref ucp_tag_recv_nb "receiver". The routine is
+ * non-blocking and therefore returns immediately, however the actual send
+ * operation may be delayed. The send operation is considered completed when
+ * it is safe to reuse the source @e buffer. If the send operation is
+ * completed immediately the routine return UCS_OK and the call-back function
+ * is @b not invoked. If the operation is @b not completed immediately
+ * and no error reported then the UCP library will schedule to invoke the
+ * call-back whenever the send operation will be completed. In other
+ * words, the completion of a message can be signaled by the return code or
+ * the call-back.
+ *
+ * @note The user should not modify any part of the @a buffer after this
+ *       operation is called, until the operation completes.
+ *
+ * @param [in]  ep          Destination endpoint handle.
+ * @param [in]  buffer      Pointer to the message buffer (payload).
+ * @param [in]  count       Number of elements to send
+ * @param [in]  tag         Message tag.
+ * @param [in]  info        Operation info, see @ref ucp_op_info_t
+ *
+ * @return NULL            - The send operation was completed immediately.
```

Comment:
> If the send operation is completed immediately the routine return UCS_OK
>  @return NULL            - The send operation was completed immediately.

Though NULL==UCS_OK==0 better to use one naming, to not confuse.

Reply:
ok, UCS_OK used